README.md

```diff
@@ -10,6 +10,21 @@
 Docker Swarm is native clustering for Docker. It turns a pool of Docker hosts
 into a single, virtual host.
 
+## Swarm Disambiguation
+
+[Swarmkit](https://github.com/docker/swarmkit): Cluster management and
+orchestration features for swarm mode in Docker Engine 1.12 or later. See the
```

Comment:
"Cluster management and orchestration features for swarm mode in Docker Engine 1.12 or later. ..." 
=>
"Cluster management and orchestration plugin in Docker Engine 1.12 or later. When using this plugin, we call the Engine running in `swarm mode` where it enables a list of [features]"